fuzz/clamav_dbload_fuzzer.cpp

```diff
@@ -117,16 +125,17 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
     FILE* fuzzdb             = NULL;
     struct cl_engine* engine = NULL;
 
-    fuzzdb = fopen(kClamAVState.tmp_db_name, "w");
+    fuzzdb = fopen(kClamAVState.tmp_db_name.c_str(), "w");
```

Comment:
Could this be `"wx"` to make sure it's impossible for two processes to write to the same file?

Reply:
Sure! I've just updated the PR with this change and in local testing it also seemed to work nicely.